evennia/contrib/auditing/example.py

```diff
@@ -0,0 +1,21 @@
+from evennia.utils.logger import log_file
```

Comment:
this file should probably not be named `example` since it actually implements a useful logging-type that many may use out of the box. Probably better to go with `default_logger.py` or somesuch.